packages/wallet/src/redux/protocols/concluding/responder/readme.md

```diff
@@ -0,0 +1,80 @@
+# Concluding Protocol
```

Comment:
I'm guessing you haven't updated this file yet?

Reply:
I thought I had...